README-Italian.md

```diff
@@ -1,110 +1,116 @@
-_The Turing Way_ overview starts at minute 5:13.
+🎧 Se preferisci un'introduzione audio al progetto, Rachael ha partecipato all'[Open Science Fair 2019](https://www.opensciencefair.eu/) (Porto, Portogallo) e la sua presentazione è disponibile su [Orion podcast](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/).
+Una panoramica di _The Turing Way_ inizia al minuto 5:13.
 
-### About the project
+### Il progetto
 
-Reproducible research is necessary to ensure that scientific work can be trusted.
-Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code.
-The goal is to ensure that all results can be independently verified and built upon in future work.
-This is sometimes easier said than done.
-Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. As these activities are not commonly taught, we recognise that the burden of requirement and new skill acquisition can be intimidating to individuals who are new to this world.
-*The Turing Way* is a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do" even for people who have never worked in this way before.
-It will include training material on version control, analysis testing, and open and transparent communication with future users, and build on Turing Institute case studies and workshops.
-This project is openly developed and any and all questions, comments and recommendations are welcome at our github repository: <https://github.com/alan-turing-institute/the-turing-way>.
+La riproducibilità della ricerca è necessaria per garantire l'affidabilità del lavoro scientifico.
+Finanziatori ed editori sempre più spesso richiedono che le pubblicazioni includano l'accesso ai dati grezzi ed al codice sorgente.
+L'obiettivo è quello di garantire che tutti i risultati siano verificabili in maniera indipendente e sviluppati nei futuri lavori.
+Tutto questo a volte è più facile a dirsi che a farsi.
+Condividere i risultati della ricerca significa comprendere la gestione dei dati, lo sviluppo del software, le scienze bibliografiche e le tecniche di integrazione continua: comptenze che non sono ben spiegate o richieste ad accademici e data scientist. Non essendo queste materie comunemente insegnate, riconosciamo che il numero di requisiti e nuove competenze richieste possano essere un problema per i neofiti.
+_The Turing Way_ è un manuale per aiutare gli studenti, i loro supervisori, finanziatori ed editori a rendere la scienza dei dati "impossibile da non fare", anche per coloro che non hanno mai lavorato secondo questi schemi.
```

Comment:
This should be
" [...] a rendere la **riproducibilita'** in data science "troppo facile da non fare".